src/libs/utils.js

```diff
@@ -89,9 +89,9 @@ export const hasAccessLevel = _.curry((required, current) => {
   return workspaceAccessLevels.indexOf(current) >= workspaceAccessLevels.indexOf(required)
 })
 
-export const canWrite = hasAccessLevel('WRITER')
-export const canRead = hasAccessLevel('READER')
-export const isOwner = hasAccessLevel('OWNER')
+export const canWrite = accessLevel => hasAccessLevel('WRITER', accessLevel)
+export const canRead = accessLevel => hasAccessLevel('READER', accessLevel)
+export const isOwner = accessLevel => hasAccessLevel('OWNER', accessLevel)
```

Comment:
Since `hasAccessLevel` is a curried function in a JS file, TS couldn't figure out that `canWrite`, etc. were callable.

Reply:
I was going to suggest breaking off this accessLevel stuff to a seperate workspace-utils.ts file and use SafeCurry2 pattern to keep the original curried exports intact... thoughts?  A followup PR maybe?  I hesitate because we are in the middle of pushing a common components re-org, and don't wanna take on a utils reorg which feels bigger.

Reply:
I'd prefer to minimize changes in this PR. I think this style is clearer and easier to understand than the original curried version, so I'm not motivated to try to keep it.